docs/getting-started.md

```diff
@@ -1,23 +1,24 @@
-# Getting Started
+---
+id: gettingStarted
```

Comment:
I noticed this `id` is used to generate the url slug for the page.

![Screenshot 2019-10-11 at 11 20 44](https://user-images.githubusercontent.com/8656188/66644585-35a73980-ec19-11e9-84d6-d2da7e864658.png)


Can we change the format to be like `getting-started` for all of the files? This will require a small change in the site config file also where we reference some pages.

Reply:
Do you need this to be done with other pages also like offlineClient to offline-client

Reply:
yes for all the pages that would be great!